kubetest/main.go

```diff
@@ -163,7 +164,7 @@ func defineFlags() *options {
 	flag.StringVar(&o.metadataSources, "metadata-sources", "images.json", "Comma-separated list of files inside ./artifacts to merge into metadata.json")
 	flag.StringVar(&o.nodeArgs, "node-args", "", "Args for node e2e tests.")
 	flag.StringVar(&o.nodeTestArgs, "node-test-args", "", "Test args specifically for node e2e tests.")
-	flag.BoolVar(&o.noAllowDup, "no-allow-dup", false, "if set --allow-dup will not be passed to push-build and --stage will error if the build already exists on the gcs path")
+	flag.BoolVar(&o.stage.noAllowDup, "no-allow-dup", false, "if set --allow-dup will not be passed to push-build and --stage will error if the build already exists on the gcs path")
```

Comment:
Relevant to the `useKrel` comment above, should we delete the `noAllowDup` field from the `options` struct?

Reply:
Hm, there are similar options like the `dockerRegistry` which are set this way, too. I thought this would be the right pattern here.

Reply:
Removed the global flag too.